docs/how-to-guides/tracing/manage.md

```diff
@@ -0,0 +1,40 @@
+# Manage traces
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+Prompt flow provides several trace toolkits in `promptflow-devkit`. This page will introduce how to delete traces in local storage with CLI/SDK.
+
+## Local trace management
+
+### Delete
+
+Prompt flow provides capability to delete traces in local storage, user can delete traces by collection, time range or prompt flow run with both CLI and SDK:
+
+::::{tab-set}
+:::{tab-item} CLI
+:sync: CLI
+
+```bash
+pf trace delete --collection <collection-name>  # delete specific collection
```

Comment:
Explain vocabulary collection?

Reply:
Also other items such as span?

Reply:
I think you can add this part in the index.md.
What items are collected? how do they organize? etc

Reply:
updated and explained collection, as it's a concept within prompt flow; for other terms like span, Honglin has one comment for a brief introduction of trace, where i attached a OTel doc on that, i think we don't need to spend spaces to explain those well known terms.